### ISSUE TITLE
Create `zemeroth_assets_src` repo

### ISSUE THREAD

**Issue by ozkriff** [ozkriff/zemeroth#192]
Create a repo to store a bunch of `.svg`s + some shell/inkscape script to autoconvert them into `.png`s.

**Comment by jupart** [comment]
If you want to upload your .svg's, I'd be happy to throw a script together to render them to .png.

**Comment by ozkriff** [comment]
Hmm, I was thinking about something very simple, like

```bash
for file in *.svg
do
    stripped=$(basename "$file" .svg)
    inkscape $filename --export-png $stripped.png
done
```

Do you think that something more advanced is required?

**Comment by jupart** [comment]
Nope! I think that'd work fine. Sometimes projects throw all their images
into a single .svg and split and convert them, but having them individually
works too. I was just looking for small things to do in the project, as
I've been keeping an eye on it since you've started it.
--

Justin Partain
